jobsdb/jobsdb_renameDs_test.go

```diff
@@ -30,9 +30,9 @@ func Test_mustRenameDS(t *testing.T) {
 
 		// And I have jobs and job status tables with events from 3 sources
 		createTables(t, dbHandle, jobsTable, jobStatusTable)
-		addJob(t, dbHandle, jobsTable, jobStatusTable, "one", "succeeded")
-		addJob(t, dbHandle, jobsTable, jobStatusTable, "two", "failed")
-		addJob(t, dbHandle, jobsTable, jobStatusTable, "three", "aborted")
+		addJob(t, dbHandle, "one", "succeeded")
```

Comment:
```suggestion
		addJob(t, dbHandle, "one", Succeeded.State)
```

I believe we can do that for the 2 ones below too. Optional of course.